java/api/src/main/java/io/ray/api/options/ActorCreationOptions.java

```diff
@@ -31,25 +31,68 @@ private ActorCreationOptions(Map<String, Double> resources, int maxRestarts,
     private String jvmOptions = null;
     private int maxConcurrency = 1;
 
+    /**
+     * Set a custom resource requirement to reserve for the lifetime of this actor.
+     * This method can be called multiple times. If the same resource is set multiple times,
+     * the latest quantity will be used.
+     *
+     * <p>By using custom resource, the user can implement virtually any actor placement policy.
```

Comment:
```suggestion
```
I'd prefer not documenting this sentence for now. Because we're working on the new placement group API. When this is done, users should use this for custom placement policy. 

The previous mechanism of custom resources is too low-level and hard to use.

Reply:
Removed